src/server/server.ts

```diff
@@ -77,17 +79,53 @@ app.all("/api/*", function (req, res) {
 });
 
 
+
+app.post("/create-chat", function(req, res) {
+  const {sender, to, text} = req.body
+  const chat = new ChatModel({
+    sender,
+    to,
+    text
+  });
+  chat
+  .save()
+  .then((data) => {
+    res.json((data))
+  })
+  .catch((err) => {
+    console.log(err);
+    res.status(501);
+    res.json({errors: err})
+  })
+})
+app.get("/chats", function(req, res) {
+  ChatModel.find()
+  .then((data) => res.json({data}))
+  .catch((err) => {
+    res.status(501);
+    res.json({ errors: err });
+  });
+})
+
 server.listen(PORT, function () {
   console.log(`starting at localhost http://localhost:${PORT}`);
 });
 
 
 io.on('connection', function(socket){
   console.log('a user connected');
```

Comment:
Your code is good, but remove the console logs before pushing for the review.

Reply:
Thanks for the suggestion. I removed it